app/helpers/application_helper.rb

```diff
@@ -2,12 +2,7 @@ module ApplicationHelper
   include Pagy::Frontend
 
   def current_service
-    case request.host
-    when ENV["CLAIMS_HOST"]
-      :claims
-    when ENV["PLACEMENTS_HOST"]
-      :placements
-    end
+    HostingEnvironment.current_service(request)
```

Comment:
I got to this too slow, but I'm not sure this logic belongs in the `HostingEnvironment` class.

I think it might be more appropriate for us to add a `Service` class/module even.

We can then add maybe `Service.extract_from_request(request)`.